Functions.Templates/Templates/RabbitMQTrigger-CSharp/readme.md

```diff
@@ -0,0 +1,11 @@
+# RabbitMQTrigger - C<span>#</span>
+
+The `RabbitMQTrigger` makes it incredibly easy to react to new events from a RabbitMQ queue. This sample demonstrates a simple use case of processing data from a given RabbitMQ Queue using C#.
+
+## How it works
+
+For a `RabbitMQTrigger` to work, you must provide a queue name that defines the queue messages will be read from.
```

Comment:
Update all read me to : `For a `RabbitMQTrigger` to work, you must provide a queue name which dictates where the queue messages should be read from.`